test/integration/hotrestart_test.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e
 
+if [[ `uname` != "Linux" ]]
```

Comment:
Yes, but I don't know how.

Normally, I could use `select` to choose either no test script or a different one. However, the output of `select` is only usable by bazel rules (because only at the point of rule execution is it possible to inspect the configuration to choose which subset of parameters to select).

`envoy_sh_test` is a macro and it attempts to access members of its `srcs` parameter to produce inputs for several rules. This doesn't work if srcs is assigned from select (whose output has no `[]` operator). To make it work, envoy_sh_test would have to become a rule and at that point I'm in over my head.